src/auth/cephx/CephxSessionHandler.cc

```diff
@@ -122,13 +119,11 @@ int CephxSessionHandler::check_message_signature(Message *m)
     // security failure, particularly when there are large numbers of
     // them, since the latter is a potential sign of an attack.  PLR
 
-    signatures_failed++;
     ldout(cct, 0) << "Signature failed." << dendl;
     return (SESSION_SIGNATURE_FAILURE);
-  }
 
-  // If we get here, the signature checked.  PLR
-  signatures_matched++;
+    // If we get here, the signature checked.  PLR
```

Comment:
this comment does not make any sense anymore without "signatures_matched++;".

please drop it.